src/components/utils/test/thread_test.cc

```diff
@@ -74,14 +78,18 @@ class ThreadTestDelegate : public threads::ThreadDelegate {
   void WaitForThreadStart() {
     AutoLock test_lock(test_mutex_);
     while (!thread_started_) {
-      cond_var_.Wait(test_lock);
+      cond_var_.WaitFor(test_lock, kMilliSecondsToWait);
```

Comment:
@orhan-mehmedov  There is only one conditional variable in the whole class. Name is not given by me, but I prefer to keep it that way, because renaming it into test_cond_var_ or something similiar would be even more obvious and redundant.